src/components/Pagination.svelte

```diff
@@ -0,0 +1,124 @@
+<script context="module">
+  export const paginateData = (array, totalPages) => {
+    let result = [];
+    let localData = array.slice();
+    for (let i = totalPages; i > 0; i -= 1) {
+      result.push(localData.splice(0, Math.ceil(localData.length / i)));
+    }
+    return result;
+  };
```

Comment:
It occurs to me that lodash's chunk would do this for us:

https://lodash.com/docs/4.17.15#chunk

(more a point of information, we don't have lodash as a dependency yet so I think we can leave using it to a followup)

Reply:
I would create a follow up issue for that if necessary.